blockbase/functions.php

```diff
@@ -27,13 +27,15 @@ function blockbase_support() {
 			)
 		);
 
-		// Register two nav menus
-		register_nav_menus(
-			array(
-				'primary' => __( 'Primary Navigation', 'blockbase' ),
-				'social' => __( 'Social Navigation', 'blockbase' )
-			)
-		);
+		// Register two nav menus if Gutenberg is activated (otherwise the __experimentalMenuLocation attribute isn't available)
+		if ( defined( 'IS_GUTENBERG_PLUGIN' ) ) {
```

Comment:
If we are keen to remove this feature but not break existing installations, could we do something like the following? 

```suggestion
		$nav_menus = get_nav_menu_locations();
		
		if ( defined( 'IS_GUTENBERG_PLUGIN' ) && ! empty( $nav_menus[ 'social' ] ) ) {
```

This way, users who have already defined menus will continue to be able to edit them via the customizer. For fresh sites with no menus defined yet, they will have to go to the site editor to edit their navigation.

Reply:
I was actually tinkering with exactly that!  However I found that the 'get_nav_menu_locations' returns the menus that are defined in the code that follows.  The 'locations' are what the theme provides.  I was not able to discern what to call to determine if the **user** had any menus defined.

I think that's a good idea for the reasons you state, though I still think that checking for Gutenberg is still important.  If that's not installed leveraging the 'classic menus' still wouldn't be supported.

Reply:
I was thinking that we would only register the menu locations if there are already menus assigned to that location, which `get_nav_menu_locations` returns just that. 

> I was not able to discern what to call to determine if the user had any menus defined.

If we wanted to make the condition for if the user had _any_ menus defined, regardless of whether they are assigned to a location, we could use https://developer.wordpress.org/reference/functions/wp_get_nav_menus/